src/commands/RunCommand.js

```diff
@@ -14,8 +16,14 @@ export const describe = "Run an npm script in each package that contains that sc
 export const builder = {
   "stream": {
     group: "Command Options:",
-    describe: "Stream output with lines prefixed by package."
-  }
+    describe: "Stream output with lines prefixed by package.",
+    type: "boolean",
```

Comment:
Discovered during the writing of new integration tests, without this explicit `type` one could cause a mangling of the arguments passed in by innocently re-ordering them. Obviously not very ideal.